install/operator/pkg/syndesis/configuration/configuration.go

```diff
@@ -112,6 +117,9 @@ var (
 
 		metaMemoryLimitFromEnv,
 		metaVolumeCapacityFromEnv,
+		sarNamespaceFromEnv,
```

Comment:
I guess this is also relevant to the sarNamespace in the templates and creeped in somehow.